README.md

```diff
@@ -1,40 +1,224 @@
-# Oh Crop! Back End
-##Get Endpoints:
+![OH Crop Logo](https://user-images.githubusercontent.com/55028065/88461332-c5903a00-ce5f-11ea-8496-d12187350486.png)
+
+## Contents
+- [About](#about)
+- [Prerequisites](#prerequisites)
+- [Getting Started](#getting-started)
+- [Installation](#installation)
+- [Endpoints](#endpoints)
+- [Running the Tests](#running-the-tests)
+- [End to End Testing](#end-to-end-testing)
+- [Built With](#built-with)
+- [Contributors](#contributors)
+
+## <a name="about"></a> About
+
+Oh Crop! Back End is an API built using Python and Flask. This API was created to
+
+### <a name="prerequisites"></a> Prerequisites
+
+You will need to have the following in order to run this app on your local machine.
+
+```
+Pip 20.1.1
+Python3 3.8.4
+Flask 1.1.2
+```
+
+## <a name="getting-started"></a> Getting Started
+
+Follow the steps below to get you a copy of the project up and running on your local machine for development and testing purposes.
+
+1. Clone this repo onto your local machine
+
+2. Run the following commands in your terminal to get the code up and running on your local machine
+
+- Make sure to run each of these without the $
+
+```
+$ pip install -r requirements.txt
+$ alembic upgrade head
+```
+
+
+### <a name="installation"></a> Installation
+
+Follow the steps below to view this app locally
+
+Step 1:
+    Start up your virtual environment by running the following command
+    - Make sure you have completed the steps in the [getting started](#getting-started) section
+```
+$ . venv/bin/activate
+```
+
+Step 2:
+    Start the local rails server by running the following command
+```
+$ flask run
+```
+
+Step 3:
+  View the website locally by visiting http://localhost:5000/
+
+  Note: All of the endpoints in the [endpoint](#endpoints) section can be run locally
+
+## <a name="endpoints"></a> Endpoints
+
+Here are some end points and request response examples to get you started.
+
+## GET Endpoints:
 __Motivational message__
+This is the home page endpoint with some sample text to ensure that the app is working as expected
+
 ```
 oh-crop-be.herokuapp.com/api/v1/
 ```
 
 __All Plants__
+This endpoint returns all plant object in the database
+
 ```
 oh-crop-be.herokuapp.com/api/v1/plants
 ```
 
 __Meet a random plant__
+This endpoint returns a random plant from the database
+
 ```
-oh-crop-be.herokuapp.com/api/v1/plants/meet
+Request:
+https://oh-crop-be.herokuapp.com/api/v1/plants/meet
+
+Response:
+{
+    "id": 15,
+    "plant_image": "https://c1.peakpx.com/wallpaper/736/669/216/appetite-avacado-avo-avocado-wallpaper-preview.jpg",
+    "plant_type": "Avocado"
 ```
 
-__Get plant by ID__
+__Get plant by ID (Plant Info)__
+This endpoint requires an ID to be passed in and it will return information about that plant
 ```
-oh-crop-be.herokuapp.com/api/v1/plants/2
+Request:
+https://oh-crop-be.herokuapp.com/api/v1/plants/2
+
+Response:
+{
+    "annual?": "Perennial",
```

Comment:
Oh, check this out!  That got changed in the DB, but never updated in the response.  Can you change this to something like annual/perennial or a/p or something like that??

Reply:
Nice catch! I'll fix that :)